### PR TITLE
v1.13.0 geoips_clavrx - VERSION RELEASE - clavrx installation steps

### PR DESCRIPTION
**Pull request by mindyls**
# Related Issues
required to close NRLMMD-GEOIPS/geoips#629
required to close NRLMMD-GEOIPS/geoips#629

# Reviewer Instructions
* Please confirm this PR is set up to merge from v1.13.0-release branch into main branch
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates (you may perform tests yourself, or review output included from another reviewer/assignee)

# Summary
Changes for version 1.13.0, uploaded with
1.13.0 update on repo geoips_clavrx.
See release note updates in 'Files changed' tab

See NRLMMD-GEOIPS/geoips#629 for additional repositories included in this update.
See NRLMMD-GEOIPS/geoips#629 for dev updates included in this update.

# Testing Instructions
See NRLMMD-GEOIPS/geoips#629 for testing instructions.

# Output
See NRLMMD-GEOIPS/geoips#629 for test output.

# Post Merge Steps
Once this PR has been merged, v1.13.0
will be tagged/released on the geoips_clavrx main branch.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,61 +1,75 @@
...
-cmap_cldMask = "geoips_clavrx.plugins.modules.colormappers.cmap_cldMask"
-cmap_cldHeight = "geoips_clavrx.plugins.modules.colormappers.cmap_cldHeight"
-cmap_cldOpd = "geoips_clavrx.plugins.modules.colormappers.cmap_cldOpd"
-cmap_cldPhase = "geoips_clavrx.plugins.modules.colormappers.cmap_cldPhase"
-cmap_cldTemp = "geoips_clavrx.plugins.modules.colormappers.cmap_cldTemp"
-cmap_IR = "geoips_clavrx.plugins.modules.colormappers.cmap_IR"
-cmap_cldReff = "geoips_clavrx.plugins.modules.colormappers.cmap_cldReff"
+
+[tool.poetry.plugins."geoips.plugin_packages"]
+geoips = "geoips_clavrx"
```
Comment (mindyls) [inline pyproject.toml:75]: I think this is wrong - shouldn't it be geoips_clavrx = "geoips_clavrx"

**Review by jsolbrig — DISMISSED** [review]

File: pyproject.toml (hunk 1 above)
Comment (jsolbrig) [reply pyproject.toml]: Yes, you are correct. I just corrected this.

**Comment by jsolbrig** [comment]
The documentation build is failing due to issues with the new, non-functional, CLAVRX install stuff. Should this be fixed before merging or just go for it?

File: geoips_clavrx/_version.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+else:
+    VERSION_TUPLE = object
+
+version: str
+__version__: str
+__version_tuple__: VERSION_TUPLE
+version_tuple: VERSION_TUPLE
+
+__version__ = version = '1.12.2a0'
+__version_tuple__ = version_tuple = (1, 12, 2)
```
Comment (mindyls) [inline geoips_clavrx/_version.py:16]: This should be 0.0.0 @jsolbrig

Comment (jsolbrig) [reply geoips_clavrx/_version.py]: This actually was an old version of the file from setuptools. I just udpated this for poetry.

**Comment by mindyls** [comment]
I am going to make an executive decision to delete all the new untested clavrx install stuff, so the doc build passes cleanly.  Might as well not have non-functional stuff in here anyway.

**Comment by mindyls** [comment]
Not going to merge this - the new installation code is not fully functional yet, and it causes the doc build to fail.  Other changes not required.

**Pull request closed without merging**